src/VisualStudio/IntegrationTest/New.IntegrationTests/VisualBasic/BasicGoToDefinition.cs

```diff
@@ -67,7 +67,7 @@ public async Task ObjectBrowserNavigation()
 
             await TestServices.SolutionExplorer.OpenFileAsync(ProjectName, "Class1.vb", HangMitigatingCancellationToken);
             await TestServices.Editor.GoToDefinitionAsync(HangMitigatingCancellationToken);
-            Assert.Equal("Int32 [from metadata]", await TestServices.Shell.GetActiveWindowCaptionAsync(HangMitigatingCancellationToken));
+            Assert.Equal("Int32 [from metadata] [Read Only]", await TestServices.Shell.GetActiveWindowCaptionAsync(HangMitigatingCancellationToken));
```

Comment:
I looked through the code in the shell and I don't fully understand why it wasn't always part of it, but my best guess is that the previous bad behaviour of setting the RDT flags, rather than combining with the existing RDT flags, might have caused some odd conditions in the editor